### PR TITLE
Update Ikea.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782

### CODE REVIEW

File: src/chrome/content/rules/Ikea.com.xml [hunk 1]
```diff
@@ -87,7 +81,6 @@ Fetch error: http://securema.ikea.com/ => https://securema.ikea.com/: (51, "SSL:
 	<target host="psathome.ikea.com" />
 	<target host="secure.ikea.com" />
 	<target host="securem.ikea.com" />
-	<target host="securema.ikea.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Ikea.com.xml]: WFM

Comment (koops76) [inline src/chrome/content/rules/Ikea.com.xml]: @J0WI https://travis-ci.org/EFForg/https-everywhere/jobs/248040734

Comment (J0WI) [inline src/chrome/content/rules/Ikea.com.xml]: They issued a new cert on 26 Jun 2017. This might have fixed the issue.

Comment (J0WI) [inline src/chrome/content/rules/Ikea.com.xml]: Well okay let's keep it excluded.

**Pull request merged**